geomesa-lambda/geomesa-lambda-tools/src/main/scala/org/locationtech/geomesa/lambda/tools/LambdaRunner.scala

```diff
@@ -19,36 +18,28 @@ import org.locationtech.geomesa.lambda.tools.data._
 import org.locationtech.geomesa.lambda.tools.export.LambdaExportCommand
 import org.locationtech.geomesa.lambda.tools.stats._
 import org.locationtech.geomesa.tools._
-import org.locationtech.geomesa.tools.export.{ConvertCommand, GenerateAvroSchemaCommand}
-import org.locationtech.geomesa.tools.status._
 
 object LambdaRunner extends RunnerWithAccumuloEnvironment {
 
   override val name: String = "geomesa-lambda"
 
-  override def createCommands(jc: JCommander): Seq[Command] = Seq(
-    new LambdaCreateSchemaCommand,
-    new LambdaDeleteFeaturesCommand,
-    new AccumuloDescribeSchemaCommand,
-    new EnvironmentCommand,
-    new AccumuloExplainCommand,
-    new LambdaExportCommand,
-    new HelpCommand(this, jc),
-    new AccumuloGetTypeNamesCommand,
-    new LambdaRemoveSchemaCommand,
-    new AccumuloVersionRemoteCommand,
-    new VersionCommand,
-    new AccumuloGetSftConfigCommand,
-    new GenerateAvroSchemaCommand,
-    new AccumuloStatsAnalyzeCommand,
-    new LambdaStatsBoundsCommand,
-    new LambdaStatsCountCommand,
-    new LambdaStatsTopKCommand,
-    new LambdaStatsHistogramCommand,
-    new AddIndexCommand,
-    new ConvertCommand,
-    new ConfigureCommand,
-    new ClasspathCommand,
-    new ScalaConsoleCommand
-  )
+  override protected def commands: Seq[Command] = {
+    super.commands ++ Seq(
+      new LambdaCreateSchemaCommand,
+      new LambdaDeleteFeaturesCommand,
+      new AccumuloDescribeSchemaCommand,
+      new AccumuloExplainCommand,
+      new LambdaExportCommand,
+      new AccumuloGetTypeNamesCommand,
+      new LambdaRemoveSchemaCommand,
+      new AccumuloVersionRemoteCommand,
```

Comment:
Probably not a major concern, but this suggests that the LDS version command only checks the version of Accumulo.  Maybe we should do something with Kafka as well?  (Or does that not make sense as there is no distributed code for Kafka?)

Reply:
yeah, there's no remote code for kafka so no remote version